hazelcast-jet-core/src/main/java/com/hazelcast/jet/pipeline/test/TestSources.java

```diff
@@ -132,6 +136,50 @@ private TestSources() {
             .build();
     }
 
+    /**
+     * Returns a {@link StreamSource} that emits an ever-increasing sequence of
+     * {@code Long} numbers with native timestamps that are exactly the same
+     * amount of time apart, as specified by the supplied {@code
+     * eventsPerSecond} parameter. The source is distributed and suitable for
+     * high-throughput performance testing. It emits the events at the maximum
+     * possible speed, constrained by the invariant that it will never emit an
+     * event whose timestamp is in the future.
+     * <p>
+     * The emission of events is distributed across the parallel processors in
+     * a round-robin fashion: processor 0 emits the first event, processor 1
+     * the second one, and so on. There is no coordination that would prevent
+     * processor 1 from emitting its event before processor 0, though, so this
+     * only applies to the event timestamps.
+     * <p>
+     * Use the {@code initialDelayMillis} parameter to give enough time to the
+     * Jet cluster to initialize the job on the whole cluster before the time
+     * of the first event arrives, so that there is no initial flood of events
+     * from the past. The point of reference is the moment at which the
+     * coordinator node creates the job's execution plan, before sending it out
+     * to the rest of the cluster.
+     * <p>
+     * This source is not fault-tolerant. The sequence will be reset once a job
+     * is restarted.
```

Comment:
We need to document:
- that the emitted value is the timestamp value
- that one has to use `withNativeTimestamps(0)` because other options are ignored (unless we fix it)
- that members should have clocks synchronized

Reply:
The actual behavior is that the emitted value is a sequence 0,1,2,...

About synchronized clocks: they should always be synchronized for member of a cluster so this doesn't create a new requirement. However, this can have a `<strong>NOTE</strong>` telling that the clock skew between any two members may result in an artificial increase of latency.

Reply:
Corresponding extension of Javadoc done w/ 680f042